### ISSUE TITLE
1.6.6 release plan

### ISSUE THREAD

**Issue by tk0miya** [sphinx-doc/sphinx#4361]
Hello maintainers,

Sorry for stopping stable releases in last months.
I caught very bad cold during December. So I could not work for Sphinx almost.
So I'd like to restart stable release cycle.

What do you think?

**Comment by tk0miya** [comment]
Note: we just start a big discussion for 1.6.x series on #4360.
If possible, I'd like to merge it to next release.

**Comment by tk0miya** [comment]
If nobody objects to my proposal, I'd like to release 1.6.6 on next weekend (Note: In Japan, Jan 8th is a National holiday and I will work for Sphinx. So it is a candidate of the X-Day).

I know #4360 is still in review. So we should pay attention for a while. Let's determine it will be merged into 1.6.6 or not on this weekend.

**Comment by tk0miya** [comment]
It seems there are no objection for X-Day. So I will release 1.6.6 in this weekend (after #4360 merged).

note: this is issues for 1.6.6 milestone. Some might be postponed to 1.6.7.
https://github.com/sphinx-doc/sphinx/milestone/39

**Comment by tk0miya** [comment]
All issues are resolved now (or postponed to 1.6.7).
So I will release it tonight (maybe 21:00-23:00 in JST)!

**Comment by tk0miya** [comment]
It's time to release!

**Comment by tk0miya** [comment]
🎉 
I just released 1.6.6
https://pypi.python.org/pypi/Sphinx

Thank you for all your work!
